tests/ClangSharp.PInvokeGenerator.UnitTests/CTest.cs

```diff
@@ -138,6 +138,28 @@ public unsafe partial struct _MyOtherUnion
     }
 }
 ";
-        return ValidateGeneratedCSharpLatestWindowsBindingsAsync(inputContents, expectedOutputContents, commandlineArgs: DefaultCClangCommandLineArgs);
+        return ValidateGeneratedCSharpLatestWindowsBindingsAsync(inputContents, expectedOutputContents, commandlineArgs: DefaultCClangCommandLineArgs, language: "c", languageStandard: DefaultCStandard);
+    }
+
+    [Test]
+    public Task MacroTest()
+    {
+        var inputContents = @"#define MyMacro1 5
+#define MyMacro2 MyMacro1";
+
+        var expectedOutputContents = @"namespace ClangSharp.Test
+{{
+    public static partial class Methods
+    {{
+        [NativeTypeName(""#define MyMacro1 5"")]
+        public const int MyMacro1 = 5;
+
+        [NativeTypeName(""#define MyMacro2 MyMacro1"")]
+        public const int MyMacro2 = 5;
+    }}
+}}
```

Comment:
Since you aren't doing interpolation:
```suggestion
        var expectedOutputContents = @"namespace ClangSharp.Test
{
    public static partial class Methods
    {
        [NativeTypeName(""#define MyMacro1 5"")]
        public const int MyMacro1 = 5;

        [NativeTypeName(""#define MyMacro2 MyMacro1"")]
        public const int MyMacro2 = 5;
    }
}
```

Reply:
Yes, sorry, I missed that